.docs/content/2.usage/use-armonik-api-in-an-angular-app.md

```diff
@@ -0,0 +1,549 @@
+```
+
+If fact, we have to use a proxy in order to redirect the gRPC call to the gRPC server (ArmoniK in our case).
+
+#### Create the proxy
+
+First, we need to create a `proxy.conf.json` file in `src` folder of our project:
+
+```json [proxy.conf.json]
+{
+  "/armonik.api.grpc.v1": {
+    "target": "http://<ip:port>", // Replace <ip:port> by the IP and port of your ArmoniK server.
+    "secure": false
+  }
+}
+```
+
+::alert{type="info"}
+We recommend to add this file to your `.gitignore` file. In fact, this file is specific to your local environment. In order to provide a template for your team, you can create a `proxy.conf.json.example` file and add it to your repository.
+::
+
+Then, we must update our config file `angular.json` in order to use the proxy:
+
+```json [angular.json]
+{
+  // ...
+  "projects": {
+    "armonik-api-angular": {
+      // ...
+      "architect": {
+        "serve": {
+          // ...
+          "options": {
+            // ...
+            "proxyConfig": "src/proxy.conf.json"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Now, we can restart our dev server:
+
+```bash
+pnpm run start
+```
+
+If you look at the console, you will see another error:
+
+```json
+{
+    "statusCode": 3,
+    "statusMessage": "Property PageSize failed validation.\nProperty Filter failed validation.\nProperty Filter failed validation.\nProperty Sort.Field failed validation.\nProperty Sort.Field failed validation.",
+    "metadata": {
+        "map": {}
+    }
+}
+```
+
+But that's a good error ! It means that we have successfully called the gRPC server.
+
+#### Fix the error
+
+For simplicity, we will update the service directly. In a real world scenario, we should pass params through the method.
+
+```diff [partitions-grpc.service.ts]
+  list$(/** You should pass params here in a real world app. */): Observable<ListPartitionsResponse> {
+-   const options = new ListPartitionsRequest()
++   const options = new ListPartitionsRequest({
++     page: 0,
++     pageSize: 10,
++     sort: {
++       direction: ListPartitionsRequest.OrderDirection.ORDER_DIRECTION_ASC,
++       field: ListPartitionsRequest.OrderByField.ORDER_BY_FIELD_ID
++     },
++     filter: {
++       id: '',
++       parentPartitionId: '',
++       podMax: 0,
++       podReserved: 0,
++       preemptionPercentage: 0,
++       priority: 0,
++     }
++   });
+
+    return this.#client.listPartitions(options);
+  }
+```
+
+And _voilà_! We have successfully called the gRPC server and displayed the result in our Angular app.
+
+You must see the loading indicator disappear and the partitions list displayed (with only a partition named `default`).
+
+#### Add the refresh button
+
+Now, we want to be able to refresh data when the user click on the refresh button. In order to do that, we will use a new subject and emit a value when the user click on the button. The merge function will then be used to
```

Comment:
Now, we want to be able to refresh data when the user clicks on the refresh button. In order to do so, we will use a new subject and emit a value when the user clicks on the button. The merge function will then be used to